llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td

```diff
@@ -2718,7 +2720,7 @@ multiclass VPseudoVGTR_VV_VX_VI<Operand ImmType = simm5, string Constraint = "">
 multiclass VPseudoVSALU_VV_VX_VI<Operand ImmType = simm5, string Constraint = ""> {
   foreach m = MxList in {
     defvar mx = m.MX;
-    defm "" : VPseudoBinaryV_VV<m, Constraint>,
+    defm "" : VPseudoBinaryV_VV<m, Constraint, Commutable=1>,
```

Comment:
Maybe add a Commutable argument to VPseudoVSALU_VV_VX_VI and explicitly set it in the PseudoVSADD[U] defms? So it's more obvious that only the add pseudos are commutable